debezium-core/pom.xml

```diff
@@ -56,6 +56,13 @@
             <scope>provided</scope>
         </dependency>
 
+        <!-- OpenLineage Integration -->
+        <dependency>
+            <groupId>io.openlineage</groupId>
+            <artifactId>openlineage-java</artifactId>
+            <scope>provided</scope>
```

Comment:
I preferred to put this in `provided` scope so that by default we will not include the `openlineage-java` dependency and also its transitive deps.